tests/test_modeling_tf_common.py

```diff
@@ -618,6 +622,75 @@ def check_encoder_attentions_output(outputs):
             self.assertEqual(model.config.output_hidden_states, True)
             check_encoder_attentions_output(outputs)
 
+    def test_headmasking(self):
```

Comment:
As this test is only for few specific models, I would better see this test inside their respective test files and remove all the occurences of `test_head_masking = False`

Reply:
I'd rather have that in a common file with `test_head_masking=False` in the other files; that's what we do for tests that are shared, and it gives the incentive of switching these to `True`.